src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/CreateCollectionCommand.java

```diff
@@ -70,6 +79,22 @@ public VectorSearchConfig(Integer dimension, String metric) {
       }
     }
 
+    public record IndexingConfig(
```

Comment:
We created a different `IndexingConfig` inside `CollectionSetting`. This is done because we can change the one inside `CollectionSetting` for usage convenience. I have allow and deny as Set in here.